tests/test_pydantic.py

```diff
@@ -248,18 +250,18 @@ class TestCallableModel:
     misc_class = MiscClass()
 
     @pytest.mark.parametrize(
-        ("fn", "args", "kwargs"),
```

Comment:
so I think there might be a difference between our formatters, we have been using ```ufmt```. Do you suggest something different? We should decide on a single formatter to use

Reply:
I've been switching to ruff wherever possible for linting, although for this PR it may have been pycharm's internal one. Ruff for everything + per-repository configs would be my preference. Their [formatter](https://github.com/astral-sh/ruff/blob/main/crates/ruff_python_formatter/README.md) is still in alpha, but mimics black style and seems to work ok.

Reply:
I will add ruff config in a separate PR later, and we can compare